Build error with UniMath.

Trying to build origin/master on my computer running OS X high sierra:

```
Messiaen ॐ  ~/Code/UniMath:(4h43m|git@master)
1731 ± sw_vers
ProductName:	Mac OS X
ProductVersion:	10.13.2
BuildVersion:	17C205
```

I get the following error when trying to run `sudo make everything` (same with just `make`):

```
COQC UniMath/CategoryTheory/All.v
File "./UniMath/CategoryTheory/All.v", line 3, characters 0-49:
Error:
Level 50 is already declared right associative while it is now expected to be left associative.

make[1]: *** [UniMath/CategoryTheory/All.vo] Error 1
make: *** [all] Error 2
```

Any ideas?

--- a/Makefile
+++ b/Makefile
@@ -391,6 +391,7 @@ UniMath/$1/All.v: UniMath/$1/.package/files
 	$(HIDE)																		\
 	exec > $$@ ;																	\
 	echo "(* This file has been auto-generated, do not edit it. *)" ;										\
+	echo "Require Export UniMath.Foundations.Init." ;												\
 	<UniMath/$1/.package/files $(FILES_FILTER_2) | grep -v '^All.v$$$$' |sed -e "s=^=Require Export UniMath.$1.=" -e "s=/=.=g" -e s/\.v$$$$/./
 endef
 $(foreach P, $(PACKAGES), $(eval $(call make-summary-file,$P)) $(eval make-summary-files: UniMath/$P/All.v))

--- a/UniMath/Algebra/All.v
+++ b/UniMath/Algebra/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.Algebra.BinaryOperations.
 Require Export UniMath.Algebra.Monoids_and_Groups.
 Require Export UniMath.Algebra.Rigs_and_Rings.

--- a/UniMath/CategoryTheory/All.v
+++ b/UniMath/CategoryTheory/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.CategoryTheory.total2_paths.
 Require Export UniMath.CategoryTheory.Categories.
 Require Export UniMath.CategoryTheory.functor_categories.

--- a/UniMath/Combinatorics/All.v
+++ b/UniMath/Combinatorics/All.v
@@ -1,7 +1,10 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.Combinatorics.StandardFiniteSets.
 Require Export UniMath.Combinatorics.Lists.
 Require Export UniMath.Combinatorics.FiniteSets.
+Require Export UniMath.Combinatorics.Graphs.
+Require Export UniMath.Combinatorics.Equivalence_Relations.
 Require Export UniMath.Combinatorics.OrderedSets.
 Require Export UniMath.Combinatorics.WellOrderedSets.
 Require Export UniMath.Combinatorics.FiniteSequences.

--- a/UniMath/Folds/All.v
+++ b/UniMath/Folds/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.Folds.UnicodeNotations.
 Require Export UniMath.Folds.aux_lemmas.
 Require Export UniMath.Folds.folds_precat.

--- a/UniMath/Foundations/All.v
+++ b/UniMath/Foundations/All.v
@@ -1,5 +1,6 @@
 (* This file has been auto-generated, do not edit it. *)
 Require Export UniMath.Foundations.Init.
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.Foundations.Preamble.
 Require Export UniMath.Foundations.PartA.
 Require Export UniMath.Foundations.PartB.

--- a/UniMath/HomologicalAlgebra/All.v
+++ b/UniMath/HomologicalAlgebra/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.HomologicalAlgebra.Triangulated.
 Require Export UniMath.HomologicalAlgebra.Complexes.
 Require Export UniMath.HomologicalAlgebra.KA.

--- a/UniMath/Ktheory/All.v
+++ b/UniMath/Ktheory/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.Ktheory.Tactics.
 Require Export UniMath.Ktheory.Equivalences.
 Require Export UniMath.Ktheory.Utilities.

--- a/UniMath/MoreFoundations/All.v
+++ b/UniMath/MoreFoundations/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.MoreFoundations.Foundations.
 Require Export UniMath.MoreFoundations.Tactics.
 Require Export UniMath.MoreFoundations.Notations.

--- a/UniMath/NumberSystems/All.v
+++ b/UniMath/NumberSystems/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.NumberSystems.NaturalNumbersAlgebra.
 Require Export UniMath.NumberSystems.NaturalNumbers_le_Inductive.
 Require Export UniMath.NumberSystems.Integers.

--- a/UniMath/PAdics/All.v
+++ b/UniMath/PAdics/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.PAdics.lemmas.
 Require Export UniMath.PAdics.fps.
 Require Export UniMath.PAdics.frac.

--- a/UniMath/RealNumbers/All.v
+++ b/UniMath/RealNumbers/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.RealNumbers.Prelim.
 Require Export UniMath.RealNumbers.Fields.
 Require Export UniMath.RealNumbers.Sets.

--- a/UniMath/SubstitutionSystems/All.v
+++ b/UniMath/SubstitutionSystems/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.SubstitutionSystems.Notation.
 Require Export UniMath.SubstitutionSystems.Signatures.
 Require Export UniMath.SubstitutionSystems.BinSumOfSignatures.

--- a/UniMath/Tactics/All.v
+++ b/UniMath/Tactics/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.Tactics.Utilities.
 Require Export UniMath.Tactics.Monoids_Tactics.
 Require Export UniMath.Tactics.Abmonoids_Tactics.

--- a/UniMath/Topology/All.v
+++ b/UniMath/Topology/All.v
@@ -1,4 +1,5 @@
 (* This file has been auto-generated, do not edit it. *)
+Require Export UniMath.Foundations.Init.
 Require Export UniMath.Topology.Prelim.
 Require Export UniMath.Topology.Filters.
 Require Export UniMath.Topology.Topology.

